pwndbg/commands/got.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import subprocess
+import struct
+
+import pwndbg.commands
+import pwndbg.commands.checksec
+import pwndbg.enhance
+import pwndbg.chain
+import pwndbg.which
+
+from pwndbg.color import *
+
+
+@pwndbg.commands.Command
+@pwndbg.commands.OnlyWhenRunning
+@pwndbg.commands.OnlyWithFile
+def got():
+    '''
```

Comment:
@degrigis This command function should more or less look like this:
```python
def got():
    '''
    Shows the state of the Global Offset Table
    '''
    sec = pwndbg.wrappers.checksec()  # i guess it can get_file on its own
    # not sure if relocs is proper name for jmpslots
    relocs = pwndbg.readelf.relocs()

    print("\nGOT protection: %s | GOT functions: %d\n " % (green(relro_status), len(relocs))

    for (address, name, got_addr) in relocs:
        print('[0x%x] %s -> %s' % (address_val, light_yellow(name), pwndbg.chain.format(got_address)))
```